tests/shared/src/test/scala/munit/ScalaCheckSeedSuite.scala

```diff
@@ -15,6 +15,26 @@ final class ScalaCheckSeedSuite extends ScalaCheckSuite {
     }
   }
 
+  test("generated int are not all the same") {
+    println(ints)
+    assert(ints.size > 1)
+  }
+}
+
+final class ScalaCheckInitialSeedSuite extends ScalaCheckSuite {
```

Comment:
Can we move this to a separate file? I normally try to organize one test suite per file.

Reply:
Fixed